### PR TITLE
Register at startup

### PR DESCRIPTION
**Pull request by cbizon**
Added a startup event that runs the kp-reload.   
Also included in that startup a registration of BTE.  It's not active right now because BTE mkg is not validating.
Refactored kps a bit to expose the right pieces for these things.
Also stripped '/' off of urls so that adding '/meta_knowledge_graph' doesn't break things (like CHP, which now loads).

### CODE REVIEW

File: kp_registry/registry.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def __init__(
 
     async def __aenter__(self):
         """Enter context."""
+        print(self.uri)
```
Comment (uhbrar) [inline kp_registry/registry.py:31]: Is this necessary, or is it a leftover from debugging?

**Review by uhbrar — COMMENTED** [review]
I'm also getting an error when running it locally: 

`sqlite3.OperationalError: unable to open database file`

However, when running it through Strider, it does function properly. Not sure why this is happening, and it might be something funky going on with permissions on my computer.

File: kp_registry/registry.py (hunk 1 above)
Comment (cbizon) [reply kp_registry/registry.py:31]: Oops, leftover

**Comment by cbizon** [comment]
On the sqlite error, I also saw that running locally via the main.sh, but not if I ran via docker-compose.  So I think it has something to do with getting PYTHONPATH right or mounting the /data directory somehow.

**Comment by cbizon** [comment]
BTE seems fixed, so I went ahead and turned that on.  I think it's ok to merge

**Review by patrickkwang — APPROVED** [review]

**Pull request merged**